CHANGELOG.md

```diff
@@ -19,6 +19,7 @@ Changelog
 
 ### Chores
 
+- Created hide show used addresses acceptance test ([PR 957](https://github.com/input-output-hk/daedalus/pull/957))
```

Comment:
When you're adding changelog entries please always add them to the bottom of the section.